### ISSUE TITLE
AttributeError: 'module' object has no attribute 'MOVPlugin'

### ISSUE THREAD

**Issue by jedisct1** [fireeye/flare-floss#241]
Currently available `floss` binary packages for Linux and OSX always fail with

```
AttributeError: 'module' object has no attribute 'MOVPlugin'
```

after the UTF-16 strings extraction.

Full traceback:
```
Traceback (most recent call last):
  File "floss/main.py", line 716, in <module>
  File "floss/main.py", line 670, in main
  File "floss/main.py", line 276, in select_plugins
  File "floss/main.py", line 110, in get_all_plugins
AttributeError: 'module' object has no attribute 'MOVPlugin'
```

**Comment by mr-tz** [comment]
Thanks! I confirmed this for the Linux nightly build and will look more into it later today.
In the meantime please use the 1.3 release available at https://github.com/fireeye/flare-floss/releases.

**Comment by mr-tz** [comment]
Fixed via #245. The nightly builds work again.
